resources/sass/app.scss

```diff
@@ -24,12 +24,7 @@ section {
 }
 
 h1, .h1 {
-    margin: $dimension-layout-xsmall 0;
     @include heading-1;
-
-    @media(min-width: $width-breakpoint-tablet){
-        margin: $dimension-layout-small 0;
-    }
```

Comment:
Why are we moving these rules from here? Should the store and the website not have the same spacing in the heading?